core/src/main/java/com/spotify/ffwd/output/CoreOutputManager.java

```diff
@@ -113,18 +118,41 @@ public long getTtl() {
     @Inject
     private Filter filter;
 
+    private HyperLogLogPlus hyperLog;
+    private AtomicLong hyperLogSwapTS;
+    private AtomicBoolean hyperLogSwapLock;
+    private Long hyperLogLogPlusSwapPeriod;
+
     public final Long getRateLimit() {
         if (rateLimiter == null) {
             return null;
         }
         return rateLimiter.getCapacity();
     }
 
+    public final Long getCardinalityLimit() {
+        if (cardinalityLimit == null) {
+            return null;
+        }
+        return cardinalityLimit;
```

Comment:
This is redundant, you can just return `cardinalityLimit` without the null check.

Reply:
removed